playbooks/roles/openconnect/templates/instructions.md.j2

```diff
@@ -13,7 +13,7 @@ OpenConnect is an extremely high-performance and lightweight VPN server that als
   * [iOS](#ios)
 
 <a name="clientcerts"></a>
-### Server and client Certifcates (macOS, Android) ###
+### Server and client Certificates (macOS, Android, iOS) ###
```

Comment:
I changed this to be specific to macOS and Android as the p12s are now embedded in the [mobileconfig](https://github.com/StreisandEffect/streisand/blob/d4e3bbc60c40e9e523f562ae91506f65582e5614/playbooks/roles/openconnect/templates/instructions.md.j2#L16).

Reply:
Fixed in b492c1a